dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -305,7 +305,7 @@ connectionHarvestTriggerCount.desc=JDBC driver property: connectionHarvestTrigge
 connectionPoolName=Connection pool name
 connectionPoolName.desc=JDBC driver property: connectionPoolName.
 connectionProperties=Connection properties
-connectionProperties.desc=JDBC driver property: connectionProperties.
+connectionProperties.desc=JDBC driver property: connectionProperties.  Encrypted values can be configured for the ssl properties javax.net.ssl.keyStorePassword and javax.net.ssl.trustStorePassword using the same process used for other data source passwords.
```

Comment:
I was copied on this because I was reviewing for Fred from the development side, and I don't agree with the request to change "JDBC driver property: connectionProperties" to "The JDBC driver property is connectionProperties".  Please look through all of the other JDBC driver description keys in this file and you will see that this change would be a departure from an existing convention that has been applied hundreds of other times within this file.  It will look very awkward if one of the JDBC driver properties strays from this.  If it is important to change it, then the change ought to be applied to every other JDBC driver property description so that all remain consistent.

Reply:
@njr-11, thanks for your feedback, and I understand. The "JDBC driver property: connectionProperties." piece is fine to leave as it is. Can the other changes still be incorporated?

Reply:
Yes, I like the other proposed changes.  Fred will need to make the update for those as the author of this pull.

Reply:
Merged change to remove extra space character, thanks for the review.

Reply:
@frowe Could you also update
"...values can be configured for the ssl properties javax.net.ssl.keyStorePassword and javax.net.ssl.trustStorePassword using the same process used for other data source passwords."
To:
"...values can be configured **for the javax.net.ssl.keyStorePassword and javax.net.ssl.trustStorePassword SSL properties by using the same process that is** used for other data source passwords."

Reply:
Done.

Reply:
@frowe Awesome, thank you!